x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engines/engines_list_indices_flyout.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiIcon,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { EnterpriseSearchEngineIndex } from '../../../../../common/types/engines';
+
+import { generateEncodedPath } from '../../../shared/encode_path_params';
+import { EuiLinkTo } from '../../../shared/react_router_helpers';
+import { SEARCH_INDEX_PATH } from '../../routes';
+import { IngestionMethod } from '../../types';
+import { ingestionMethodToText } from '../../utils/indices';
+
+import { EnginesListLogic } from './engines_list_logic';
+
+const healthColorsMap = {
```

Comment:
`healthColorsMap` is [duplicated a few times](https://github.com/elastic/kibana/search?q=healthColorsMap) across the Enterprise Search plugin. Is it time to make this a shared constant?

Reply:
I noticed same. What about moving the `healthColorsMap` somewhere in [common](https://github.com/elastic/kibana/tree/5df280162515c9429cbd172a56eba2baace6baf1/x-pack/plugins/enterprise_search/common) folder, WDYT? @sphilipse

Reply:
Making this shared makes a lot of sense, but you'll want to put this in [public/applications/shared](https://github.com/elastic/kibana/tree/5df280162515c9429cbd172a56eba2baace6baf1/x-pack/plugins/enterprise_search/public/applications/shared) instead. `common` is meant for things that are shared across the server and the frontend. `public/applications/shared` is meant for things that are shared across the different frontends.

Reply:
@sphilipse moved  `healthColorsMap`  to [public/applications/shared](https://github.com/elastic/kibana/tree/5df280162515c9429cbd172a56eba2baace6baf1/x-pack/plugins/enterprise_search/public/applications/shared) in this [PR](https://github.com/elastic/kibana/pull/149271)